cron.8

```diff
@@ -22,9 +22,11 @@
 cron \- daemon to execute scheduled commands (Vixie Cron)
 .SH SYNOPSIS
 .B cron
-.RB [ \-l
-.IR load_avg ]
+.RB [ \-m
```

Comment:
FreeBSD cron uses -m to set the default MAILTO, perhaps -M could be used here instead?